easybuild/easyconfigs/c/Caffe/Caffe-1.0-intel-2017a-Python-2.7.13.eb

```diff
@@ -0,0 +1,67 @@
+# -*- mode: python; -*-
+# EasyBuild reciPY for Caffe as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2017 UL HPC
+# Authors::   UL HPC Team <hpc-sysadmins@uni.lu>
+# License::   MIT/GPL
+#
+easyblock = 'CMakeMake'
+
+name = 'Caffe'
+version = '1.0'
+
+# cudaversion = '7.5.18'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/BVLC/caffe'
+description = """
+Caffe is a deep learning framework made with expression, speed,
+and modularity in mind. It is developed by the Berkeley Vision
+and Learning Center (BVLC) and community contributors.
+"""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+# toolchainopts = {'optarch': True, 'pic': True}
```

Comment:
Why have you commented this out?

Reply:
Didn't look relevant to the intel toolchain (was originaly part of the `foss`recipe).